utils_cv/tracking/model.py

```diff
@@ -236,7 +236,80 @@ def save(self, path) -> None:
 
         save_model(path, self.epoch, self.model, self.optimizer)
         print(f"Model saved to {path}")
+                        
+    def savetxt_results(self,
+                        results: Dict[int, List[TrackingBbox]],
+                        exp_name: str = 'results',
+                        root_path: str = None,
+                        result_filename: str = 'results.txt',
+                        if_tmp: bool = True) -> str:
+        """Save tracking results to txt in tmp directory or provided path.
+        
+        Args:
+            results: prediction results from predict() function, i.e. Dict[int, List[TrackingBbox]]
+            exp_name: subfolder for each experiment
+            root_path: results saved root path. Default: None
```

Comment:
Instead of root_path, results_filename and if_tmp can we just have a single variable called e.g. result_file_path. By default this variable is initialized to None. If the user over-writes it, then it's used, otherwise a temporary directory and name "result.txt" is used. That solves the problem that some inputs get ignored if "is_tmp" is set to Trye.

Reply:
The following line: if if_tmp or not root_path: serves the purpose you have mentioned above with the default values.

Have to keep result_filename when running multiple experiments with mot challenge, because each experiment needs to assign a unique name.